README.md

```diff
@@ -80,17 +88,17 @@ output "aws_parameters" {
 
 ### Inputs explained
 
-| Name                      | Description                                                     | Type     | Default             | Required |
-| --------------------------- | ----------------------------------------------------------------- | ---------- | --------------------- | ---------- |
-| integration_name          | Prefix to be used for naming new resources                      | `string` | `UptycsIntegration` |          |
-| upt_account_id            | Uptycs AWS account ID                                           | `string` | `""`                | Yes      |
-| aws_account_id            | AWS organization's master account ID                            | `string` | `""`                | Yes      |
-| external_id               | External ID                                                     | `uuid4`  | `""`                | Yes      |
-| vpc_flowlogs_bucket_name  | Name of the S3 bucket in master for VPC flow logs               | `string` | `""`                | Optional |
-| cloudtrail_s3_bucket_name | Name of the organization cloud trail S3 bucket                  | `string` | `""`                | Optional |
-| kinesis_stream_name       | Name of the organization Kinesis stream                         | `string` | `""`                | Optional |
-
 
+| Name                      | Description                                                         | Type     | Default             | Required |
+| --------------------------- | --------------------------------------------------------------------- | ---------- | --------------------- | ---------- |
+| integration_name          | Prefix to be used for naming new resources                          | `string` | `UptycsIntegration` |          |
+| upt_account_id            | Uptycs AWS account ID                                               | `string` | `""`                | Yes      |
+| aws_account_id            | AWS organization's master account ID                                | `string` | `""`                | Yes      |
+| external_id               | External ID                                                         | `uuid4`  | `""`                | Yes      |
+| vpc_flowlogs_bucket_name  | Name of the S3 bucket in master for VPC flow logs                   | `string` | `""`                | Optional |
+| cloudtrail_s3_bucket_name | Name of the organization cloud trail S3 bucket                      | `string` | `""`                | Optional |
+| kinesis_stream_name       | Name of the organization Kinesis stream                             | `string` | `""`                | Optional |
+| cloudtrail_in_master     | Specifies whether the cloudtrail set up is in master account or not | `bool` |     `true`                | Yes      |
```

Comment:
@llukalapu-uptycs It cannot be "required" it we are suggesting a default value (like integration-name)